cinema/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+from cinema.views import *
```

Comment:
It's better to import only the views you need explicitly instead of using a wildcard import. This improves code readability and avoids potential conflicts.